### PR TITLE
Feature - create New note

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Feature - Add New Note 
Login using demo - button
Preview - https://notes-app-git-new-note-vishnu-aithal.vercel.app/

### CODE REVIEW

File: src/components/NoteEditor.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                            noteDetails.color === "blue"
+                                ? "border-2 scale-125 shadow-md border-zinc-600 dark:border-slate-200"
+                                : ""
+                        }`}></button>
+                    <button
+                        onClick={() => dispatch(setColor("default"))}
+                        className={`h-5 w-5 border rounded-md mx-1 ${
+                            noteDetails.color === "default"
+                                ? "border-2 scale-125 shadow-md border-zinc-600 dark:border-slate-200"
+                                : ""
```
Comment (ApurvaSawant11) [inline src/components/NoteEditor.js:165]: Here, everything is the same except the `color`. See if you can use a map function to reduce redundancy.

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me! You can merge the PR

File: src/components/NoteEditor.js (hunk 1 above)
Comment (Vishnu-Aithal) [reply src/components/NoteEditor.js:165]: yes will do , thanks for reviewing

**Review by abhisheksahani-nova — APPROVED** [review]
Looks clean and subtle , good to merge

**Pull request merged**